### PR TITLE
Adding support for sqs queues in different accounts

### PR DESCRIPTION
**Pull request by jsabo**
This should enable beaver to log to sqs queues in another account.

### CODE REVIEW

**Comment by josegonzalez** [comment]
Don't the key and secret handle this already?

**Comment by jsabo** [comment]
http://boto.readthedocs.org/en/latest/ref/sqs.html?highlight=sqs#boto.sqs.connection.SQSConnection.get_queue

I don't think so no.  It's an extra arg to get_queue.   I'm also not putting the key and secret key on disk and using roles for that.

File: beaver/transports/sqs_transport.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ def __init__(self, beaver_config, logger=None):
         self._secret_key = beaver_config.get('sqs_aws_secret_key')
         self._region = beaver_config.get('sqs_aws_region')
         self._queue_name = beaver_config.get('sqs_aws_queue')
+        self._queue_owner_acct_id = beaver_config.get('sqs_aws_queue_owner_acct_id')
```
Comment (josegonzalez) [inline beaver/transports/sqs_transport.py]: Can you add this to the config.py, as well as the docs?

**Comment by josegonzalez** [comment]
Seems you can write to an sqs queue if you are not the owner...

File: beaver/transports/sqs_transport.py (hunk 1 above)
Comment (jsabo) [inline beaver/transports/sqs_transport.py]: Sure.  One second.

**Comment by jsabo** [comment]
Yeah you can white list random accounts.

image

**Pull request merged**